frontend/javascripts/oxalis/model/sagas/isosurface_saga.js

```diff
@@ -296,14 +321,50 @@ function* removeIsosurface(action: RemoveIsosurfaceAction): Saga<void> {
   }
 }
 
+function* noteEveryEditedCell(): Saga<void> {
+  const activeCellId = yield* select(state => enforceVolumeTracing(state.tracing).activeCellId);
+  modifiedCells.add(activeCellId);
+}
+
+function* refreshIsosurfaces(): Saga<void> {
+  const renderIsosurfaces = yield* select(state => state.datasetConfiguration.renderIsosurfaces);
+  if (!renderIsosurfaces) {
+    return;
+  }
+  yield* put(saveNowAction());
+  // We reload all cells that got modified till the start of reloading.
+  // By that we avoid that removing cells that got annotated during reloading from the modifiedCells set.
+  const currentlyModifiedCells = new Set(modifiedCells);
+  modifiedCells.clear();
+  // First create an array containing information about all loaded isosurfaces as the map is manipulated within the loop.
+  for (const [cellId, threeDMap] of Array.from(isosurfacesMap.entries())) {
+    if (!currentlyModifiedCells.has(cellId)) {
+      continue;
+    }
+    const isoSurfacePositions = threeDMap.entries().filter(([value, _position]) => value);
+    if (isoSurfacePositions.length <= 0) {
```

Comment:
```suggestion
    if (isoSurfacePositions.length === 0) {
```

or can it be negative? ^^